lib/process-services-cloud/src/lib/process/process-filters/components/process-filter-dialog-cloud.component.html

```diff
@@ -6,12 +6,12 @@
         <mat-card-content class="adf-process-filter-dialog-card-content">
         <form [formGroup]="filterForm">
             <mat-form-field class="adf-process-filter-name" [floatLabel]="'auto'">
-                <input matInput placeholder="{{ 'ADF_CLOUD_EDIT_PROCESS_FILTER.FILTER_NAME' | translate }}" formControlName="name" id="adf-filter-name-id">
+                <input matInput placeholder="{{ 'ADF_CLOUD_EDIT_PROCESS_FILTER.FILTER_NAME' | translate }}" [formControl]="filterForm.controls.name" id="adf-filter-name-id">
```

Comment:
I will wrap **filterForm.controls.name** as a getter **filterName**, would be easier to understand but the change is up to you, approved